tfhe/src/core_crypto/algorithms/slice_algorithms.rs

```diff
@@ -301,3 +301,11 @@ where
     lhs.iter_mut()
         .for_each(|lhs| *lhs = (*lhs).wrapping_mul(rhs));
 }
+
+pub fn slice_wrapping_scalar_div_assign<Scalar>(lhs: &mut [Scalar], rhs: Scalar)
```

Comment:
what is this used for? if i recall correctly, division is not a necessarily a constant-time operation, so it may be leaking info if not used properly

Reply:
scaling during decryption if we use a torus smaller than the native torus

Reply:
or during GGSW encryption of a given factor but it's not dependant on the secret key content normally

Reply:
we have other algorithms that are not constant time (like pk encryption at the moment)